tests/validation/tests/Engine/fixtures.py

```diff
@@ -96,3 +97,9 @@ def test_time(request):
     if test_time is None:
         return 30
     return int(test_time)
+
+
+@pytest.fixture(autouse=True)
+def delay_between_tests():
```

Comment:
why this delay ?

Reply:
We discussed this on validation meeting - tests are being run to fast one after another and tend to fail.